picdar-export/src/main/scala/com.gu.mediaservice.picdarexport/lib/picdar/PicdarApi.scala

```diff
@@ -90,27 +94,34 @@ trait PicdarApi extends HttpClient with PicdarInterface with LogHelper {
     val lastIndex = math.min(start + length, searchInstance.count)
     val searchItemsFuture = for {
       response   <- post(messages.retrieveResults(mak, searchInstance.id, firstIndex, lastIndex))
-      searchItems = response \ "ResponseData" \ "Match"
+      searchItems = response \ "ResponseData" \ "Record"
     } yield searchItems
 
     searchItemsFuture map { searchItems =>
       for {
         matchNode  <- searchItems
         urn         = (matchNode \ "MMRef" text)
-        dateLoaded  = extractField(matchNode, "Date Loaded") map picdarAmericanDateFormat.parseDateTime getOrElse
-          (throw new Error("Failed to read date loaded from search result"))
+        _ = println(matchNode)
+        dateString <- extractField(matchNode, "Date Loaded")
+
+        // Where we're going Marty - we don't need accurate dates
+        dateLoaded  = picdarAmericanDateFormat.parseDateTime("19551105")
```

Comment:
The date loaded in this context is used solely for breaking up requests to `Picdar` into day ranges - `fakepicdar` has no such restriction on search set size so i'm using this date as a key to run operations against the set of images from `fakepicdar`.